docs/guides/on-demand.mdx

```diff
@@ -299,103 +296,38 @@ const uploadAsset = async () => {
 }
 ```
 
-2. Upload the asset to the generated URL
-
-Get the “url”: “https://origin.livepeer.studio/api/assets/…“ from the response.
-
-Check the status of the generated url.
-
-https://livepeer.studio/api/asset/$ASSET_ID/ with provided asset.id
+### Upload the video
```

Comment:
I don't think this section is consistent with the overall organization here. The first step of the tutorial is not a section itself, just an enumeration `1.`. Then the request/response are `###` sections themselves, then this is another section on the same level. 

I think we should either have:
 - request/response as `####` sections and then we consistenly create a `###` section for each step in the tutorial
 - do not have this `###` section here. The step is already described in the `2.` enumeration below.